### ISSUE TITLE
Value order is modified after squishing

### ISSUE THREAD

**Issue by DerekYu177** [DerekYu177/dynamictableprint#8]
Before printing to screen: 
![image](https://user-images.githubusercontent.com/11148779/42413001-880655e8-81e5-11e8-9c7b-fdabcc65176e.png)

After: 
![image](https://user-images.githubusercontent.com/11148779/42413004-91318eda-81e5-11e8-97e2-8e873f376d8d.png)

**Comment by DerekYu177** [comment]
This can be solved by freezing the indices right after creation. The relevant modification location should be somewhere around https://github.com/DerekYu177/dynamictableprint/blob/master/dynamictableprint/dynamicprinter.py#L54:

```py
```

**Comment by DerekYu177** [comment]
Closed since this issue is fixed in 09b866becbc16c37deb3256a99e5ade9b4774480

**Comment by DerekYu177** [comment]
This is actually a much bigger issue than we think. Since https://github.com/DerekYu177/dynamictableprint/blob/master/dynamictableprint/squisher.py#L45 we have:

```py
self._sdf.loc[index, column] = self._squish_to(value, ideal_length)
```

It means that we depend on having the columns _in order_. If they are not, this looks like it will randomly cause mutations in the dataset. Thank goodness it has been fixed.
